appinventor/docs/markdown/reference/components/media.md

```diff
@@ -376,6 +376,12 @@ None
 
 {:.properties}
 
+{:id="SpeechRecognizer.Language" .text .bo} *Language*
+: Suggests the language to use for recognizing speech. An empty string (the default) will
+ use the system's default language.
+
+     Language is specified using an IETF language tag, such as en-US or es-MX.
```

Comment:
I think the documentation should link to a list of these codes, since most users are going to have to Google their code anyway.